CHANGELOG.md

```diff
@@ -8,6 +8,138 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
+    - `WordPress.Functions.FunctionRestrictions` - use the `AbstractFunctionRestrictionsSniff` class instead.
+    - `WordPress.Variables.VariableRestrictions` without replacement.
+- The following sniffs which were previously deprecated, have been removed:
+    - `WordPress.Arrays.ArrayDeclaration` - use the other sniffs in the `WordPress.Arrays` category instead.
+    - `WordPress.CSRF.NonceVerification` - use `WordPress.Security.NonceVerification` instead.
+    - `WordPress.Functions.DontExtract` - use `WordPress.PHP.DontExtract` instead.
+    - `WordPress.Variables.GlobalVariables` - use `WordPress.WP.GlobalVariablesOverride` instead.
+    - `WordPress.VIP.CronInterval` - use `WordPress.WP.CronInterval` instead.
+    - `WordPress.VIP.DirectDatabaseQuery` - use `WordPress.DB.DirectDatabaseQuery` instead.
+    - `WordPress.VIP.PluginMenuSlug` - use `WordPress.Security.PluginMenuSlug` instead.
+    - `WordPress.VIP.SlowDBQuery` - use `WordPress.DB.SlowDBQuery` instead.
+    - `WordPress.VIP.TimezoneChange` - use `WordPress.WP.TimezoneChange` instead.
+    - `WordPress.VIP.ValidatedSanitizedInput` - use `WordPress.Security.ValidatedSanitizedInput` instead.
+    - `WordPress.WP.PreparedSQL` - use `WordPress.DB.PreparedSQL` instead.
+    - `WordPress.XSS.EscapeOutput` - use `WordPress.Security.EscapeOutput` instead.
+    - `WordPress.PHP.DiscouragedFunctions` without direct replacement.
+        The checks previously contained in this sniff were moved to separate sniffs in WPCS 0.11.0.
+    - `WordPress.Variables.VariableRestrictions` without replacement.
+    - `WordPress.VIP.AdminBarRemoval` without replacement.
+    - `WordPress.VIP.FileSystemWritesDisallow` without replacement.
+    - `WordPress.VIP.OrderByRand` without replacement.
+    - `WordPress.VIP.PostsPerPage` without replacement.
+        Part of the previous functionality was split off in WPCS 1.0.0 to the `WordPress.WP.PostsPerPage` sniff.
+    - `WordPress.VIP.RestrictedFunctions` without replacement.
+    - `WordPress.VIP.RestrictedVariables` without replacement.
+    - `WordPress.VIP.SessionFunctionsUsage` without replacement.
+    - `WordPress.VIP.SessionVariableUsage` without replacement.
+    - `WordPress.VIP.SuperGlobalInputUsage` without replacement.
+- The `WordPress.DB.SlowDBQuery.DeprecatedWhitelistFlagFound` error code which is superseded by the blanket deprecation warning for using the now deprecated WPCS native whitelist comments.
+- The `WordPress.PHP.TypeCasts.NonLowercaseFound` error code which has been replaced by the upstream `Generic.PHP.LowerCaseType` sniff.
+- The `WordPress.PHP.TypeCasts.LongBoolFound` and `WordPress.PHP.TypeCasts.LongIntFound` error codes which has been replaced by the new upstream `PSR12.Keywords.ShortFormTypeKeywords` sniff.
+- The `WordPress.Security.EscapeOutput.OutputNotEscapedShortEcho` error code which was only ever used if WPCS would be run on PHP 5.3 with the `short_open_tag` ini directive set to `off`.
```

Comment:
In if clauses that refer to something that did not happen in the past, the first verb should be in the past tense (were run) instead of the conditional (would be run).